farm/data_handler/input_features.py

```diff
@@ -410,6 +410,77 @@ def sample_to_features_qa(sample, tokenizer, max_seq_len, sp_toks_start, sp_toks
                     "seq_2_start_t": seq_2_start_t}
     return [feature_dict]
 
+def sample_to_features_dpr(
+    sample, max_seq_len, query_tokenizer, passage_tokenizer, embed_title=True, remove_sep_tok_from_untitled_passages=True
```

Comment:
remove `remove_sep_tok_from_untitled_passages`

Reply:
removed it completely